### DISCUSSION TITLE
Contrasting wiring diagrams

### DISCUSSION THREAD

**Discussion by nmt28** [STELLA-Landsat/STELLA#55] [Debugging]
The wiring diagram on the Stella 1.1 webpage is different from that in the .zip folder. There is a small horizontal blue wire in two places (sorry don't have the numbers with me - see photos). Which is correct?

![20240314_180127.jpg](https://github.com/STELLA-Landsat/STELLA/assets/22138389/e8ad3307-f11c-44b3-a169-149c02e22449)

![20240314_180155.jpg](https://github.com/STELLA-Landsat/STELLA/assets/22138389/04745430-e604-4810-8046-875902835c2c)

**Comment by Landsat-Mike** [comment]
I am not seeing the discrepancy. If it's the small blue wire connects from H38 to H41 in both images. I also checked the downloads for the wire and it is H38 to H41 as well. The longer blue wire connects from F41 to F51.

**Reply by nmt28** [reply]
Apologies, I uploaded the same image twice. See new images below:

![20240314_180155.jpg](https://github.com/STELLA-Landsat/STELLA/assets/22138389/aebcf1b2-d5d3-4dc3-bfb2-1eb8375f40d3)

![20240314_180211.jpg](https://github.com/STELLA-Landsat/STELLA/assets/22138389/dbe502c8-abbc-42e4-afff-40f7c4b89a8b)

**Reply by nmt28** [reply]
Although, I'm wondering if it even matters. The breadboard pins run vertically so should it matter, provided the numbers match?
